package.json

```diff
@@ -0,0 +1,14 @@
+{
+  "name": "daydream",
+  "packageManager": "yarn@3.2.4",
+  "private": true,
+  "workspaces": [
+    "common",
+    "web",
+    "mobile"
+  ],
+  "scripts": {
+    "start": "yarn workspace @daydream/mobile start",
+    "dev": "yarn workspace @daydream/web dev"
```

Comment:
We might not need these scripts. We can just cd into the directory and run the correct project. Also not sure how yarn workspaces handles this way of running the project